AirSkyBoat/AirSkyBoat
# Summoner Duration/Range Fixes

**_I affirm:_**

## What does this pull request do?
- Reduced range of Assault to 18
- Duration of sleepga is supposed to be static 90 seconds
- Fixed stat drain from Ultimate Terror


## Steps to test these changes
use these pet abilities and check if it drains and sleeps for the fixed duration
